django_usaid_clinicfinder/clinicfinder/tasks.py

```diff
@@ -299,6 +313,25 @@ def run(self, lookuppointofinterest_id, **kwargs):
                  via Celery.',
                 exc_info=True)
 
+    def format_match_aat(self, match):
+        return "%s (%s)" % (
+            match.get('OrganisationName'),
+            match.get('FullAddress'))
+
+    def run_att(self, lookuppoi):
+        url = (
+            "%(url)s?username=%(username)s&password=%(password)s&meters=50000"
+            "&category=77&x=%(x)s&y=%(y)s") % {
```

Comment:
We will need to set the category based on the kind of request made (i.e. we can't hardcode it to `77` like we do here at the moment).